test/comparison_to_analytic_solutions.jl

```diff
@@ -93,27 +92,26 @@ struct DummyImpurityDensity{T} <: SolidStateDetectors.AbstractImpurityDensity{T}
     function SolidStateDetectors.get_impurity_density(cdm::DummyImpurityDensity{T}, pt::CartesianPoint{T})::T where {T <: SSDFloat}
         SolidStateDetectors.get_impurity_density(cdm, CylindricalPoint(pt))
     end
-
     sim_cyl.detector = SolidStateDetector(sim_cyl.detector, DummyImpurityDensity{T}());
     sim_car.detector = SolidStateDetector(sim_car.detector, DummyImpurityDensity{T}());
 
-    calculate_electric_potential!(sim_cyl, 
-        max_refinements = 4
+    calculate_electric_potential!(sim_cyl, grid = Grid(sim_cyl, max_tick_distance = (1u"cm", 15u"°", 1u"m")),
+        convergence_limit = 1e-7, refinement_limits = [0.2, 0.1, 0.05], use_nthreads = 1, verbose = false
     )
-    calculate_electric_potential!(sim_car, 
-        max_refinements = 1
+    calculate_electric_potential!(sim_car, grid = Grid(sim_car, max_tick_distance = (0.3u"cm", 0.3u"cm", 1u"m")),
+        convergence_limit = 1e-6, refinement_limits = [0.2, 0.1, 0.05, 0.01], use_nthreads = 1, verbose = false
     )
 
-    idxR1 = searchsortedfirst( sim_cyl.electric_potential.grid.axes[1], ustrip(R1))
-    rs = sim_cyl.electric_potential.grid.axes[1].ticks[idxR1:end]
-    potential_rms_cyl = sqrt(sum((ustrip.(map(r -> potential_analytic(r * u"m"), rs)) .- sim_cyl.electric_potential.data[idxR1:end, 1, 1]).^2))
-    idxR1 = searchsortedfirst( sim_car.electric_potential.grid.axes[1], ustrip(R1))
-    rs = sim_car.electric_potential.grid.axes[1].ticks[idxR1:end]
-    idy0 = searchsortedfirst( sim_car.electric_potential.grid.axes[2], 0)
-    potential_rms_car = sqrt(sum((ustrip.(map(r -> potential_analytic(r * u"m"), rs)) .- sim_car.electric_potential.data[idxR1:end, idy0, 1]).^2))
+    idxR1 = searchsortedfirst( sim_cyl.electric_potential.grid.axes[1], ustrip(R1));
+    rs = sim_cyl.electric_potential.grid.axes[1].ticks[idxR1:end];
+    potential_rms_cyl = sqrt(sum((ustrip.(map(r -> potential_analytic(r * u"m"), rs)) .- sim_cyl.electric_potential.data[idxR1:end, 1, 1]).^2)) / (ustrip(BV_true)*length(rs));
+    idxR1 = searchsortedfirst( sim_car.electric_potential.grid.axes[1], ustrip(R1));
+    rs = sim_car.electric_potential.grid.axes[1].ticks[idxR1:end];
+    idy0 = searchsortedfirst( sim_car.electric_potential.grid.axes[2], 0);
+    potential_rms_car = sqrt(sum((ustrip.(map(r -> potential_analytic(r * u"m"), rs)) .- sim_car.electric_potential.data[idxR1:end, idy0, 1]).^2))/ (ustrip(BV_true)*length(rs));
 
     @testset "Potential RMS" begin
-        @test potential_rms_cyl < 0.01
-        @test potential_rms_car < 1.1
+        @test potential_rms_cyl < 1e-3
+        @test potential_rms_car < 1e-3
```

Comment:
Why did this change so drastically?

Reply:
It is now normalized to the bias voltage and number of ticks.